Lesson Plans/Flight Instructor/CFI/CFI1.4 Teaching Process.md

```diff
@@ -0,0 +1,250 @@
+---
+tags:
+  - cfi
+  - lesson
+---
+Related to [[Steps in the Teaching Process]] and [[Elements of the Teaching Process]]
+
+### Objective
+The learner acquires knowledge and skills related to course & lesson development for aviation instruction.
+
+### Resources
+- [[CFI ACS]] Section FI.I.C
+- [[AIH Ch5]]
+- [[AIH Ch7]]
+
+### Builds On
+None
+
+### Schedule
+Ground 0.5 hours
+
+### Lesson Elements
+
+#### Teaching
+1. Process
+	1. Organizes the material in a way that the learner can understand
+	2. Preparation, Presentation, Application and Assesment
+2. Essential Teaching Skills
+	1. People Skills
+		1. Technical knowledge is useless if not communicated effectively
+		2. Effective communication is two-way and includes active listening to the learner
+		3. Challenge the learner while supporting their efforts
+	2. Subject Matter Expertise
+		1. Effective instructor have a sincere interesting in learning & professional growth
+		2. Networking and observing other instructors and seeking mentoring is also helpful; Be a lifelong learner
+	3. Management Skills
+		1. Ability to plan, organize, lead and supervise (a lesson)
+		2. Manage the time effectively, plan for the unexpected
+		3. Aviation instructor also need to supervise learners (ex. preflight procedures)
+	4. Assessment Skills
+		1. Measuring the progress (behavior change)
+		2. Purpose of the assessment should be made clear
+
+#### [[Flight Instructor Model Code of Conduct]]
+
+#### Course of Training
+1. Commonly referred to as Curriculum, syllabus or lesson plan
+2. Curriculum: multiple courses for various pilot certificates and ratings
+3. Syllabus: Summary or outline of an individual course of study that includes multiple lessons
+4. Lesson plan: Detailed plan for how a specific lesson will be conducted, including lesson objectives, organization of materials, description of teaching aids, instructions and learner actions, and evaluation criteria, completion standards
+
+#### Preparation of a Lesson
+1. A lesson plan must precedes instruction
+2. Training Objectives & Standards
+	1. Types of Objectives
+		1. Performance-based: Define exactly what needs to be done and how it is done
+		2. Decision-based: Use more dynamic environment and [[Scenario-Based Training|SBT]] to teach critical thinking skills, such as [[RM]] and [[ADM]]
+	2. Standards
+		1. Closely tied to objectives
+		2. It should also include comprehensive examples; more difficult in affective domain (attitudes, beliefs and value)
+3. Performance-based Objectives
+	1. Set **measurable**, reasonable standards
+	2. Includes: description, condition and criteria
+		1. Description: Desired outcome of knowledge or skills
+			1. Condition: Rules for demonstration of the skill (equipment, reference, limitations, etc...)
+		2. Criteria: Standards that measure the accomplishment of the objective
```

Comment:
This is one of the trickier parts of flight instruction, IMO. It feels like the PPL lessons should have 'gates' on when you proceed to the next lesson, but since each person learns differently, sometimes people really benefit from moving quickly. Others really need to stick to a lesson until they can basically pass the checkride. It would be amazing to find a way to slice this problem differently so it could be a single criteria that applies to everyone...

anyway, the checkride is a proper gate, so I guess as long as it works earlier thats good enough ¯\_(ツ)_/¯